packages/untp-test-suite/src/interfaces/cli/test.ts

```diff
@@ -14,16 +15,20 @@ test
     `Configuration file (default path: "${defaultCredentialFilePath}")`,
     defaultCredentialFilePath,
   )
-  .action(async (options) => {
+  .action( (options) => {
     try {
       let credentialPath = `${process.cwd()}/${credentialFileName}`;
       if (options.config) {
         credentialPath = path.resolve(process.cwd(), options.config);
       }
 
-      const testSuiteResults = await processTestSuite(credentialPath);
+      // const testSuiteResults = await processTestSuite(credentialPath);
 
-      console.log(JSON.stringify(testSuiteResults, null, 2));
+      // const testSuiteMessage = getLogStatus(testSuiteResult.testSuiteResults);
+      // const testSuiteFinalReport = getFinalReport(testSuiteResult);
+
+      // console.log(testSuiteMessage);
+      // console.log(testSuiteFinalReport);
```

Comment:
Why do you comment out the code?

Reply:
The `getLogStatus` function and the `getFinalReport` function now use the new interfaces defined in PR [#31](https://github.com/uncefact/tests-untp/pull/31). If you look at line 25, it calls the `processTestSuite` function, which in this branch is outdated and its data is not correct to pass to the `getLogStatus` function and the `getFinalReport` function. This means that this PR has a slight dependency on PR [#31](https://github.com/uncefact/tests-untp/pull/31). If you want me to remove the comments here, I can mock data and pass that to the `getLogStatus` function and the `getFinalReport` function for testing purposes.